pke/kyber/templates/pkg.templ.go

```diff
@@ -133,6 +152,17 @@ func (pk *PublicKey) Unpack(buf []byte) {
 	(*internal.PublicKey)(pk).Unpack(buf)
 }
 
+// Unpacks pk from the given buffer.
+//
+// Returns an error if the buffer is not of the right size, or the public
+// key is not normalized.
+func (pk *PublicKey) UnpackMLKEM(buf []byte) error {
+	if len(buf) != PublicKeySize {
+		return kem.ErrPubKeySize
```

Comment:
The previous `Pack/Unpack` function panics instead of returning an error when the size does not match.